packages/xo-web/src/xo-app/logs/backup-ng/log-alert-header.js

```diff
@@ -81,6 +84,11 @@ export default decorate([
             </Button>
           </CopyToClipboard>
         </Tooltip>
+        <Tooltip content={_('logDownload')}>
+          <Button size='small' onClick={effects._downloadLog}>
+            <Icon icon='reply' />
```

Comment:
Use a download icon.

Reply:
Have you a recommandation, because i don't find any download icon in `xo-icon-*` ?

Reply:
If you don't find what you need, you can add it.
```scss
&-download {
  @extend .fa;
  @extend .fa-download;
}
```
around [there](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/icons.scss#L350).